projects/developer/src/app/navbar/navbar.component.scss

```diff
@@ -8,6 +8,105 @@ $navbar-item-width: 150px;
 
 .navbar-ctr {
     height: $navbar-height;
+    position: relative;
+}
+
+.navbar-ctr--with-banner {
+    margin-top: $navbar-height;
+}
+
+.navbar-banner {
+    position: fixed;
+    height: $navbar-height;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    width: 100%;
+    background: var(--info-main);
+    color: var(--on-info-main);
+    z-index: 10000;
+}
+
+.banner-message {
+    flex-grow: 1;
+    padding-left: 1.5rem;
+    font-weight: 400;
+    font-size: 1rem;
+}
+
+.banner-icon {
+    font-size: 2.3rem;
+    padding-top: 1.1rem;
+    background: var(--info-second);
+    color: var(--on-second);
+    width: 5rem;
+    height: $navbar-height;
+    text-align: center;
+}
+
+.banner-close {
+    font-size: 1.2rem;
+    padding: 1.2rem;
+    background: var(--info-main);
+    color: var(--on-info-main);
+    width: 5rem !important;
+    height: $navbar-height;
+    border: 0;
+    &:hover {
+         background-color: var(--info-second) !important;
+         color: var(--on-second) !important;
+    }
+}
+
+.navbar-banner-success {

```

Comment:
not a big deal, but i think'd you be able to structure these differently. something like:

```css
.navbar-banner-success {
  // properties here

  .banner-icon { }
  .banner-close {}
}
```

basically just the .banner-icon-<type> and .banner-close-<type> could go away. no need to fix though, just for your awareness.

Reply:
I can fix it; it would be more elegant and I always like elegant!